### DISCUSSION TITLE
unable to add to vcenter

### DISCUSSION THREAD

**Discussion by Hr46ph** [w2c/letsencrypt-esxi#28] [General]
vcenter removed the esx server from the cluster and I am unable to add it back.

I keep getting:
  Authenticity of the host's SSL certificate is not verified.

I tested DNS lookups and reverse lookups from the vcenter and that just works fine.
I changed the certificate mode to custom and tried thumbprint as well, neither works.
I added the certificate chain for lets encrypt to the certificate manager, didnt help either.

I had to remove the app and revert to self signed from the vmware ca.

Does this sound familiar? I read the wiki troubleshooting and searched issues and discussions but couldn't find anything related.

The error I receive is plenty to be found all over the net, but the solutions don't seem to work.

If we can't solve this, you might want to add a big fat warning somewhere because once a server is removed, all its settings are removed as well (think monitoring, alerts, all host specific settings basically are gone.

**Comment by Churro** [comment]
In the readme file of `w2c-letsencrypt-esxi`, the very first sentence says that this solution is tailored for _standalone_ VMware ESXi servers. I have never tested it with vCenter but in case you find a reliable way to make it work, I'd be happy to add instructions to the wiki.

**Reply by Hr46ph** [reply]
Sure, and it wasn't meant to critique the tool itself, on which you did a great job!
